index.html

```diff
@@ -184,62 +185,96 @@
 
 - TFS supports Git and mixing Git/TFVC repos in the same project.
 - Git is the default version control provider for new projects in Azure DevOps (Online & On-Prem).
-- Microsoft advises using Git over TFVC.
 - Integrated Git support in Visual Studio and Visual Studio Code.
   - You will want to learn some basic commands.
+- Comparison table showing that Git can do everything that TFVC can.
+  - [Comparison of Git and TFVC capabilities](https://docs.microsoft.com/en-us/azure/devops/repos/tfvc/comparison-git-tfvc?view=azure-devops)
+---
+
+## Migrating to Git
 
-- [Comparison of Git and TFVC capabilities](https://docs.microsoft.com/en-us/azure/devops/repos/tfvc/comparison-git-tfvc?view=azure-devops)
+### Keep your history in the past!
+
+- Microsoft suggests doing **tip migrations**.
+  - Bring only a snapshot of the most recent version of a branch into Git.
+--
+
+- Why?
+  - Lossy process, at best.
+  - Possibly misleading.
+  - High level of effort involved.
+  - Relative rarity usage of the history.
+--
+
+- Where *do* you keep the history then?
+  - In your old version control system!
+  - Refer to your old repositories in your README.md files.
+---
+
+## Migrating to Git
+
+### Keep your history in the past!
```

Comment:
You've doubled up on these headers, I'm not sure if that is intentional.
(see line 196)

Reply:
That *was* the idea, but it was just a cheap trick instead of correctly ordering my content. Removed in the latest version.